examples/triton/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '2.1'
+# Prometheus demonstration of the autopilot pattern
+
+services:
+  prometheus:
+    image: autopilotpattern/prometheus:${TAG:-latest}
+    # Joyent recommends setting instances to always restart on Triton
+    restart: always
+    labels:
+      # This label sets the CNS name, Triton's automatic DNS
+      # Learn more at https://docs.joyent.com/public-cloud/network/cns
+      - triton.cns.services=prometheus
+      # This label selects the proper Joyent resource package
+      # https://www.joyent.com/blog/optimizing-docker-on-triton#ram-cpu-and-disk-resources-for-your-containers
+      - com.joyent.package=g4-highcpu-1G
+    network_mode: bridge
+    ports:
+      # You may not want these port declarations for production. Without them, Prometheus will only
+      # listen on the private network. This will also result in a public prometheus CNS record being created,
+      # in the triton.zone domain.
+      - 9090
+    env_file: _env
+    environment:
+      - CONSUL_AGENT=1
+      - CONSUL=pc.svc.${TRITON_CNS_SEARCH_DOMAIN_PRIVATE}
+
+
+  # Consul is the service catalog
+  consul:
+    image: autopilotpattern/consul:0.7.2-r0.8
```

Comment:
We'll need to update this one (https://github.com/autopilotpattern/consul/releases), but we can do that later: https://github.com/autopilotpattern/prometheus/issues/22